lib/OpenQA/Worker/Job.pm

```diff
@@ -455,7 +459,8 @@ sub _stop_step_4_upload ($self, $reason, $callback) {
                 # replace some file names
                 my $ofile = $file;
                 $ofile =~ s/serial0/serial0.txt/;
-                $ofile =~ s/virtio_console.log/serial_terminal.txt/;
+                $ofile =~ s/virtio_console(.*)\.log/serial_terminal$1.txt/;
+                $ofile =~ s/\.log/.txt/;
```

Comment:
As this is a more intrusive change I suggest you create a separate PR to make the change more explicit.

Reply:
what I meant is to simply split it into its own PR without further changes so that we can merge it independently, like one PR tomorrow and another the day after that :)

Reply:
Ok, I checked the change in detail and likely this will effectively really only impact virtio_console files so it's good as is.